diff -urN xfce4-session-4.10.0/scripts/xflock4 xfce4-session-4.10.0_slimlock/scripts/xflock4
--- xfce4-session-4.10.0/scripts/xflock4	2012-04-29 05:43:27.000000000 +0900
+++ xfce4-session-4.10.0_slimlock/scripts/xflock4	2012-06-29 15:48:04.122894494 +0900
@@ -35,7 +35,8 @@
 # else run another access locking utility, if installed
 for lock_cmd in \
   "xlock -mode blank" \
-  "slock"
+  "slock" \
+  "slimlock"
   do
     set -- $lock_cmd
     if command -v -- $1 >/dev/null 2>&1; then
